api/v1alpha1/openapi.yaml

```diff
@@ -1090,33 +1161,97 @@ components:
           format: uuid
         name:
           type: string
-        createdAt:
+        sourceType:
           type: string
-          format: date-time
-        inventory:
-          $ref: '#/components/schemas/Inventory'
-        sourceID:
+          enum: [inventory, rvtools, source]
+        source_id:
           type: string
           format: uuid
+        createdAt:
+          type: string
+          format: date-time
+        snapshots:
+          type: array
+          items:
+            $ref: '#/components/schemas/Snapshot'
+        labels:
+          type: array
+          items:
+            $ref: '#/components/schemas/Label'
       required:
         - id
         - name
+        - source
         - createdAt
-        - inventory
-        - sourceID
+        - snapshots
```

Comment:
Should snapshot be required? There could be assessment without snapshots IMHO

Reply:
yes. it's impossible. An assessment is a series of inventories.
> There could be assessment without snapshots IMHO
How? The only situation is when the user wants to create an assessment from a source without inventory. Which to be hones, it does not make sense for me